### DISCUSSION TITLE
Project files not uploading to qfieldcloud

### DISCUSSION THREAD

**Discussion by shapley** [opengisch/QField#2456] [Q&A]
I have attempted to upload a project from desktop QGIS to the QFieldCloud. I made the cloud version the definitive version and in my local directory i can see that there are these files (working on Win10):-

local dir = QField\cloud\my_new_project\.qfieldsync\backup...download...upload
export dir = QField\export\my_new_project\my_project_name_qfield.qgs...data.gpkg

However no files appear on QFieldCloud 'project' folder when i'm logged in. 

I am also wondering whether I should tick 'Is public' in the cloud project settings?

I'm not sure where it went wrong or why it has failed to upload and hope someone can help.
Are there any logs I can check? Sorry... I am a new 'beta' subscriber.

**Comment by suricactus** [comment]
Hello @shapley.

Thanks for trying QFieldCloud.

> However no files appear on QFieldCloud 'project' folder when i'm logged in.

Not exactly sure what you mean there, but let me try to explain. Once you have created a QFieldCloud project (a project that you see both in QFieldSync and QFieldCloud website), you need to add the files. You already have set the local directory, which means QFieldSync will try to upload these files to the cloud if you ask the plugin to do so.

![image](https://user-images.githubusercontent.com/2820439/136626018-34d629a0-07a3-43ba-a781-39b9935cbb04.png)

After you select your project in the project list, you can see all buttons at the bottom toolbar are enabled. You need to click the second button (cloud with arrows) and you can transfer your files to the cloud. Hope this works.

> I am also wondering whether I should tick 'Is public' in the cloud project settings?

In most of the cases you don't want to make your project public, because everyone with QFieldCloud account can see and read your project.

**Reply by shapley** [reply]
Hi Suricactus,

The Projects 'Overview' screen is showing 'The locally stored cloud project is already synchronised with QFieldCloud, no action is required' suggesting that there should be files in the project directory. When i log into 'https://app.qfield.cloud/a/' I s
![capture](https://user-images.githubusercontent.com/7022885/136753248-84da832f-4938-4657-9192-69a122839ccb.PNG)
![folder](https://user-images.githubusercontent.com/7022885/136753250-84bfff43-401c-4136-8c3e-4ff97491654c.PNG)
ee

**Reply by suricactus** [reply]
Hi @shapley . I see. Can you please share your settings of the cloud project? On QFieldSync please double click on the project of interest. Please there you should see "Local Directory". It should hold the directory path to your project. Make sure you have **all** your project files in that directory. 

![image](https://user-images.githubusercontent.com/2820439/136757095-95c472d7-3efd-4956-8877-1f980db6ceef.png)

**Reply by shapley** [reply]
Hi Suricactus,

Here are some images. It looks like it is working and files appear to be uploading but no files appear on 'Cloud. Maybe a 'network firewall' issue? Perhaps there is a log I can send.
![settings4](https://user-images.githubusercontent.com/7022885/136801496-2c91fb90-b9fc-485f-a9d6-80885d734584.PNG)
![settings3](https://user-images.githubusercontent.com/7022885/136801500-d0f28fd8-1c68-4c9e-8c97-39c9fdfd2936.PNG)
![settings2](https://user-images.githubusercontent.com/7022885/136801501-dae13250-f873-4c65-999a-15121fb56dbd.PNG)
![settings1](https://user-images.githubusercontent.com/7022885/136801503-66c1f219-768c-4b2e-b368-c5c33f0851bb.PNG)
'

**Reply by suricactus** [reply]
Can you also screenshot your directory contents of your Local Directory:
![image](https://user-images.githubusercontent.com/2820439/136804097-c9c1f536-f295-4f9e-ae35-bd5b219e8b73.png)

These is where you should have all the files you want to keep on the cloud.

**Reply by shapley** [reply]
Hi Suricactus, 

Please see belo
![localdir](https://user-images.githubusercontent.com/7022885/136907798-5dad304d-5c88-4b7a-824d-f5d4209c4b00.PNG)
w:

**Reply by suricactus** [reply]
Hello, @shapley.

Thanks for the screenshot.

You need  to put all your projec files withing this directory and they you will be able to sync. In your initial message you said you have a .gpkg file which I don't see on the screenshot. The `.qgs` file ends in `.qgs~` which is a temporary copy, QFieldSync will ignore that one too. So you need to copy the real .qgs file too.

**Reply by shapley** [reply]
Hi Suricactus,

I have done it this time...I had to create a new project in the QFieldCloud QGIS plugin then accept the default settings on the 'synchronising' tab (not click on the cloud options which showed the 'red' bins) then the files uploaded. Many thanks for your patienc
![leave-default](https://user-images.githubusercontent.com/7022885/137151444-86118be5-8a8a-4823-b892-4fbc09f055ff.PNG)
e. Please close this call.
![leave-success](https://user-images.githubusercontent.com/7022885/137151466-9a6b3234-1f4d-441e-9ef2-241532dd22b8.PNG)
.

**Reply by suricactus** [reply]
Great! I hope you would enjoy using QFieldCloud and feel free to report new issues or ideas to improve the service.

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).
